### ISSUE TITLE
Suggestion: Ability to delete audio book

### ISSUE THREAD

**Issue by lonchat** [PaulWoitaschek/MaterialAudiobookPlayer#455]
Hi, I really like the clean design of your Audio Book player. I listen to podcasts and would like to be able to delete a podcast when I have listened to it, from the player, rather than having to open up a file manager.

Hope this can be included.

Thanks

James

**Comment by nitro322** [comment]
Not sure what the protocol is for voting on issues, but I'd really like to see this feature as well.  Normally for books it's not a big deal because I can pop over to a file manager afterwards and delete the (obvious) book, but I've also had an issue when listening to podcasts.  I'm not sure what order it users to display podcasts, but it doesn't appear to be alphabetical, or at least doesn't follow the same rules as my file manager.  On two separate occasions I listened to, eg., 1 of 2 podcasts, switched to the file manager to delete the "first" podcast, then switched back to antennapod only to discover that I deleted the one I hadn't finished.  Being able to directly deleted would be a safeguard as well as convenience.

Separately, just in case you're wondering why I'm using an audio book player for podcasts... I use a separate podcast player for most podcasts, but every now and then I'll find a random episode or two of something I don't subscribe to that I want to listen to, and it's far easier to copy that to the phone and listen with material player than temporarily subscribing to the podcast feed, listening, then unsubscribing from the feed when I'm done.  Probably not a common use case, but it's very handy.
